draft-ietf-acme-acme.md

```diff
@@ -922,11 +934,11 @@ Content-Type: application/jose+json
 {
   "protected": base64url({
     "alg": "ES256",
-    "jwk": {...},
```

Comment:
It says above that you need a "jwk" for this request :)  It looks like you intend for the client to put the key in the "key" field, but I would be a little more comfortable having the CA assemble the object.  Basically for the same reason we are using "kid" in general -- because it shows forces the CA to process the thing.